src/main/java/org/radarbase/management/web/rest/ProjectLiteResource.kt

```diff
@@ -0,0 +1,43 @@
+package org.radarbase.management.web.rest
+
+import io.micrometer.core.annotation.Timed
+import org.radarbase.management.service.ProjectService
+import org.radarbase.management.web.rest.util.PaginationUtil
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.data.domain.Pageable
+import org.springframework.data.web.PageableDefault
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+/**
+ * GET  /lite-projects : get all the projects.
+ *
+ * @return the ResponseEntity with status 200 (OK) and the list of ProjectLiteDTO
+ */
+
+@RestController
+@RequestMapping("/api")
+class ProjectLiteResource(
```

Comment:
You can call this `PublicResource` with annotation `@RequestMapping("/api/public")` and then we can house all the public endpoints in the future here.